packages/@uppy/svelte/package.json

```
@@ -1,20 +1,11 @@
 {
   "name": "@uppy/svelte",
+  "version": "3.1.3",
```

That doesn't look right

---

Merge conflict gone wrong.

```suggestion
  "version": "4.0.0-beta.1",
```